### ISSUE TITLE
More options within the pitch bend.

### ISSUE THREAD

**Issue by Gabrielxd195** [LMMS/lmms#5613]
### Summary of improvements
Add a menu with the options of: velocity, panning, pitch bend within the pitch bend editor on the piano roll.

#### Justification
With the pitch bend editor I can work with piano roll notes individually, and it would be very useful to add a menu with the options of: velocity, pan, pitch bend within a menu. Currently this can be done with separate automation tracks but this affects the workflow. but with this idea implemented for example: it would make a violin sound more realistic, and it would help us for many other things.

#### Sketch

![Mas opciones dentro del pitch bend recortado](https://user-images.githubusercontent.com/26127693/89475339-db520900-d755-11ea-9de0-87e08aad5621.png)

**Comment by Spekular** [comment]
Automating velocity doesn't make sense. Velocity represents the speed at which a key is pressed, and as such cannot change over the course of a note. I'd be surprised if the MIDI spec supported changing the velocity of a note over time, and even more so if any major instruments supported it.

Per-note detune automation is also different from automating the pitch knob, as it applies to individual notes rather than the entire clip. I imagine supporting per-note pan would be a significant amount of work, so to be clear this isn't just the case of exposing existing features via a new UI.

**Comment by Gabrielxd195** [comment]
> Automating velocity doesn't make sense. Velocity represents the speed at which a key is pressed, and as such cannot change over the course of a note. I'd be surprised if the MIDI spec supported changing the velocity of a note over time, and even more so if any major instruments supported it.
> 
You are right, I confused speed with volume. but I think the concept of what I'm trying to explain is very clear.

> Per-note detune automation is also different from automating the pitch knob, as it applies to individual notes rather than the entire clip. I imagine supporting per-note pan would be a significant amount of work, so to be clear this isn't just the case of exposing existing features via a new UI.

The pattern automation system is different from the note automation system, because the latter automates the notes individually. and it would be useful to have the possibility of automating the volume of the note, because although you can do this by automating the pattern, it is less practical. also in this case, there would only be a single gui element, which would be the menu where you choose to automate the detune, volume and pan.

I'm going to change the photo to avoid confusion.
